1704581563742	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1704581699040	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1704581699040	FirefoxAccounts	DEBUG	getOAuthToken enter
1704581699042	FirefoxAccounts	TRACE	getCachedToken returning cached token
1704581699042	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1704581699044	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1704581699047	Services.Common.RESTRequest	DEBUG	GET request to https://api.accounts.firefox.com/v1/account/attached_clients
1704581742070	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1704581742070	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.sys.mjs :: onStopRequest :: line 479\"  data: no]"}
1704581742072	Services.Common.RESTRequest	DEBUG	GET https://api.accounts.firefox.com/v1/account/attached_clients failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1704581742072	Hawk	WARN	hawk request error: [Exception... "NS_ERROR_UNKNOWN_HOST"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: onStopRequest :: line 479"  data: no] Stack trace: onStopRequest()@resource://services-common/rest.sys.mjs:479
1704581742073	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: get status :: line 616"  data: no] Stack trace: get status()@resource://services-common/rest.sys.mjs:616
request()@resource://services-common/hawkclient.sys.mjs:230
1704581742073	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status text: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: get statusText :: line 631"  data: no] Stack trace: get statusText()@resource://services-common/rest.sys.mjs:631
request()@resource://services-common/hawkclient.sys.mjs:238
1704581742073	Hawk	DEBUG	(Response) /account/attached_clients: code: null - Status text: null
1704581742073	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: get headers :: line 663"  data: no] Stack trace: get headers()@resource://services-common/rest.sys.mjs:663
_maybeNotifyBackoff()@resource://services-common/hawkclient.sys.mjs:307
request()@resource://services-common/hawkclient.sys.mjs:246
1704581742074	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: get headers :: line 663"  data: no] Stack trace: get headers()@resource://services-common/rest.sys.mjs:663
_maybeNotifyBackoff()@resource://services-common/hawkclient.sys.mjs:307
request()@resource://services-common/hawkclient.sys.mjs:247
1704581742074	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status text: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: get statusText :: line 631"  data: no] Stack trace: get statusText()@resource://services-common/rest.sys.mjs:631
_constructError()@resource://services-common/hawkclient.sys.mjs:113
request()@resource://services-common/hawkclient.sys.mjs:252
1704581742074	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: get status :: line 616"  data: no] Stack trace: get status()@resource://services-common/rest.sys.mjs:616
_constructError()@resource://services-common/hawkclient.sys.mjs:114
request()@resource://services-common/hawkclient.sys.mjs:252
1704581742074	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: get status :: line 616"  data: no] Stack trace: get status()@resource://services-common/rest.sys.mjs:616
_constructError()@resource://services-common/hawkclient.sys.mjs:115
request()@resource://services-common/hawkclient.sys.mjs:252
1704581742074	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.sys.mjs :: get headers :: line 663"  data: no] Stack trace: get headers()@resource://services-common/rest.sys.mjs:663
_constructError()@resource://services-common/hawkclient.sys.mjs:121
request()@resource://services-common/hawkclient.sys.mjs:252
1704581742075	FirefoxAccounts	ERROR	error GETing /account/attached_clients: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.sys.mjs :: onStopRequest :: line 479\"  data: no]","message":null,"code":null,"errno":null}
1704581744867	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1704581744963	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1704581744963	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1704581744966	Sync.Service	DEBUG	User-Agent: Firefox/121.0 (Intel Mac OS X 10.15) FxSync/1.123.0.20231211174248.desktop
1704581744966	Sync.Service	INFO	Starting sync at 2024-01-06 22:55:44 in browser session goNU7d8pvLmN
1704581744967	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1704581744969	Sync.Status	INFO	Resetting Status.
1704581744969	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1704581744970	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1704581756439	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1704581756593	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1704581756593	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1704581756595	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1704581756597	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/197306892/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1704581756597	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1704581756597	Sync.Status	DEBUG	Status.service: success.status_ok => error.sync.failed
1704581756597	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1704581756597	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=reschedule)
1704581756598	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
